apps/api/Dockerfile

```diff
@@ -23,7 +23,7 @@ FROM pnpm AS installer
 WORKDIR /app
 
 COPY --from=builder /app/out/json/ .
-COPY --from=builder /app/out/pnpm-lock.yaml ./pnpm-lock.yaml
+COPY pnpm-lock.yaml ./pnpm-lock.yaml
```

Comment:
Why are we no longer using pruned lock file?

Reply:
because sometimes turbo incorrectly prunes the file and remove some files we need (e.g. mongoose and shit) https://github.com/vercel/turbo/issues/2791

Reply:
i figure it's fine if we use the full lockfile, since we pruned the package json anyways?

Reply:
Huh, it looks fine over here. No missing modules.

Reply:
can you try using pruned pnpm-lock and build reg-scraper image?